inyoka_theme_default/templates/portal/usercp/change_password.html

```diff
@@ -0,0 +1,59 @@
+{#
+    portal/usercp/change_password.html
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    This page dispalys a form where the user can change his password.
+
+    :copyright: (c) 2007-2015 by the Inyoka Team, see AUTHORS for more details.
+    :license: BSD, see LICENSE for more details.
+#}
+
+{% extends 'portal/usercp/overall.html' %}
+{% import 'macros.html' as macros %}
+
+{% block breadcrumb %}
+  {{ super() }}
+
+  {{ macros.breadcrumb_item(_('Change password'), href('portal', 'usercp', 'password'))}}
+{% endblock %}
+{% set selected = 'password' %}
+
+{% block content %}
+  <h3>{% trans %}Change password{% endtrans %}</h3>
+  {% trans %}
+    You can change your password with this form. For security reasons, you need to enter your old password.
+  {% endtrans %}
+  {% call macros.outer_form(csrf_token(), form, manually_rendered=True,submit_label="Change password") %}
```

Comment:
* whitespace before `submit_label`? (ok, thats not the biggest bug ever found, but…)
 * gettext for translation of "Change password"?